.github/workflows/ci_workflows.yml

```diff
@@ -47,7 +47,12 @@ jobs:
   tests:
     needs: [initial_checks]
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
+    secrets:
+      CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
     with:
+      setenv: |
+        ARCH_ON_CI: ${{ env.ARCH_ON_CI }}
```

Comment:
Now GHA is complaining that this syntax is invalid: https://github.com/astropy/astropy/actions/runs/4420300294

*The workflow is not valid. .github/workflows/ci_workflows.yml (Line: 53, Col: 15): Unrecognized named-value: 'env'. Located at position 1 within expression: env.ARCH_ON_CI*

Reply:
I found https://github.com/orgs/community/discussions/26529 but all the proposed solutions look too painful. Maybe I'll just copy-paste and move on.

Reply:
I hadn't realised env couldn't be used for inputs! You could add an extra step to initial_check to make them a job output, but I think the neatest way would be copy-paste